source/tools/administration-interfaces.txt

```diff
@@ -432,17 +432,21 @@ and includes the following features:
 
 - <http://3t.io/mongochef/>
 
-3T MongoChef is a multi-platform, in-place data browser and editor,
-and includes the following features:
-
-- Edit any MongoDB data type directly in a tree-view, table-view or in a lossless JSON editor.
-- Edit complex structures such as embedded documents or arrays-in-arrays-in-embedded-documents in both a tree-view or table-view effortlessly.
-- See live evaluation of your data as you type (very handy for data types such as Date, Timestamp, etc).
-- Seamlessly step into embedded objects, either for all documents simultaneously or for just individual documents.
-- Build JSON queries with simple drag-and-drop.
-- Copy & paste documents across databases and collections.
-- Import & export documents from and to different databases, collections, and JSON files.
-- Multi-platform: Mac OS X, Windows, Linux.
+3T MongoChef is a MongoDB GUI for ``Mac``, ``Windows``, and ``Linux``.
+Full support for ``MongoDB 2.4``, ``MongoDB 2.6``, and ``MongoDB 3.0``.
+
+- ``In-place editing`` inside your tree-view, table-view or in a lossless JSON editor
```

Comment:
Double backticks here and elsewhere will be interpreted as casting to fixed-width font in RST - see http://docutils.sourceforge.net/docs/user/rst/quickref.html